Ariadne/src/Ariadne/Config/Presence.hs

```diff
@@ -0,0 +1,34 @@
+module Ariadne.Config.Presence
```

Comment:
This file is `Ariadne/src/Ariadne/Config/Presence.hs`, but should be `ariadne/src/Ariadne/Config/Presence.hs`

Reply:
Sorry, I renamed the folder when I met a warning from `cabal`.